### ISSUE TITLE
Langfuse Integration: start/end time of generators are not logged

### ISSUE THREAD

**Issue by marcklingen** [deepset-ai/haystack-core-integrations#1262]
**Describe the bug**

While this works for non-LLM/generator nodes, generator nodes do not log start/end times to Langfuse.

Example trace: https://cloud.langfuse.com/project/cloramnkj0002jz088vzn1ja4/traces/42cb3022-b114-4e33-b8c7-774229d451b6?observation=9326286a-fdd4-4f12-8fe0-b22e480df767&display=details

**To Reproduce**

https://github.com/langfuse/langfuse-docs/blob/main/cookbook/integration_haystack.ipynb

**Describe your environment (please complete the following information):**

Colab with latest packages, see above.

**Comment by vblagoje** [comment]
Thanks for logging this bug, @marcklingen. Just to confirm: once this bug is fixed, the Langfuse web UI will show how long it took to generate the response—similar to the other components in the screenshot, right?
<img width="448" alt="Image" src="https://github.com/user-attachments/assets/bbdf2f4b-0817-4d08-b86d-8c501157c763" />

**Comment by marcklingen** [comment]
That's correct @vblagoje! This currently works for all components except for LLM/generator nodes

**Comment by vblagoje** [comment]
Fixed and included in https://pypi.org/project/langfuse-haystack/0.7.0/ release
